config.json

```diff
@@ -866,9 +866,11 @@
       "slug": "connect",
       "core": false,
       "unlocked_by": null,
-      "difficulty": 1,
+      "difficulty": 9,
       "topics": [
-
+        "Control-flow (loops)",
+        "Games",
+        "Graphs"
```

Comment:
I was thinking the board really ends up being a straight-up [graph](https://en.wikipedia.org/wiki/Graph_(abstract_data_type)) you need to traverse.